welcomescreen.py

```diff
@@ -73,26 +72,28 @@ def make(self, gameDisplay, back):
             # gameicon=pygame.image.load('data/images/icon.png')
             # pygame.display.set_icon(gameicon)
 
+        # Functions sx(coord, shift) and sy(coord) are defined in rules.py
+        # sx() and sy() are used to scale appropriately
```

Comment:
Should I add the sx() sy() documentation here or somewhere else? @quozl

Reply:
We usually put any documentation about functions in a docstring at the start of the function, and rely on explicit import to locate the functions.  You've used wildcard imports, which make that a bit unusual, but it was the style before you began.

Reply:
Yes, the wildcard imports make it slightly difficult to locate the function definitions, so I thought of adding the comments. Maybe I should add these comments as docstrings right after `class` statement?

Reply:
You could not use wildcard imports for these functions?  Explicit imports are better overall, and makes flake8 and pep8 happier.

Reply:
Done in b0656a6